rust/repositories.bzl

```diff
@@ -409,7 +417,7 @@ def load_arbitrary_tool(ctx, tool_name, tool_subdirectories, version, iso_date,
 
     This function sources the tool from the Rust-lang static file server. The index is available
     at: https://static.rust-lang.org/dist/index.html (or the path specified by
-    "${RUST_STATIC_URL}/dist/index.html" if the RUST_STATIC_URL envinronment variable is set).
+    "${STATIC_RUST_URL}/dist/index.html" if the STATIC_RUST_URL envinronment variable is set).
```

Comment:
I'm not sure it's the best idea to change the environment variable name. Whats the justification here?

Reply:
i think only the docs were wrong as some lines below the variable "STATIC_RUST_URL" is imported from environment.

Reply:
Ah, good catch!